tests/e2e/specs/ont/ont_runs_view.cy.js

```diff
@@ -23,4 +24,35 @@ describe('ONT Runs view', () => {
     cy.get('#created_at').invoke('text').should('match', /\d+/)
     cy.get('#actions').invoke('text').should('include', 'Edit').and('include', 'Sample Sheet')
   })
+
+  it('displays the ONT run page with correct data when clicking on edit run', () => {
+    cy.wrap(OntRunsFactory()).as('ontRunsFactory')
+    cy.wrap(OntRunFactory()).as('ontRunFactory')
+
+    cy.get('@ontRunsFactory').then((ontRunsFactory) => {
+      cy.intercept('GET', '/v1/ont/runs?page[size]=25&page[number]=1&include=instrument', {
+        statusCode: 200,
+        body: ontRunsFactory.content,
+      })
+    })
+
+    cy.get('@ontRunFactory').then((ontRunFactory) => {
+      cy.intercept('GET', '/v1/ont/runs/2?include=flowcells', {
+        statusCode: 200,
+        body: ontRunFactory.content,
+      })
+    })
+
+    cy.intercept('/v1/ont/pools?include=tube,libraries.tag,libraries.request', {
+      fixture: 'tractionOntPools.json',
+    })
+
+    cy.visit('#/ont/runs')
+    cy.get('#editRun-2').click()
+
+    // Check that the URL is correct
+    cy.url().should('include', '#/ont/run/2')
+    cy.get('#flowcell-id-1').invoke('val').should('eq', 'ABC1234')
```

Comment:
Advisory. We should probably change this so it relates to the store data but I suggest we do that at another time and get this story deployed.

Reply:
Ok 👍